modules/local/validation/metadata/main.nf

```diff
@@ -0,0 +1,135 @@
+process VALIDATION_METADATA {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/python:3.9--1' :
+        'biocontainers/python:3.9--1' }"
+
+    input:
+    tuple val(meta), path(payload), path(payload_files), path(specimen), path(sample), path(experiment), path(read_group)
```

Comment:
Expected input channel is
```
tuple val(meta), tuple val(clinical),tuple val(analysis)
```
such that a specimen would be:
```
path(clinical.specimen)
```

can either change this code or massage channel to fit this requirement

Reply:
Yes, had code to massage the outputs from upstream channels to generate the input channel requirements:
https://github.com/Pan-Canadian-Genome-Library/molecular-data-submission-workflow/blob/integrate_test/workflows/molecular-data-submission-workflow.nf#L79-L110